<?xml version="1.0"?>

<launch>
  <!--PARAMETERS-->
  <param name="ellipse_scale" value="1"/>
  <param name="rawlog_play_delay" value="0.1"/>
  <param name="ini_filename" value="$(find mrpt_ekf_slam_3d)/tutorial/kf-slam_6D_demo.ini"/>
  <param name="rawlog_filename" value="$(find mrpt_ekf_slam_3d)/tutorial/kf-slam_6D_demo.rawlog"/>
  <param name="odom_frame_id" value="/odom"/>
  <param name="global_frame_id" value="/map"/>
  <param name="base_frame_id" value="/base_link"/>
  <!-- for multiple sensors, place comma eg "r1/front_laser/scan,/scan_rear " -->
  <param name="sensor_source" value="/front_laser/scan"/>

  <!-- Setting environment variables -->
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find mrpt_ekf_slam_3d)/config/rosconsole.config"/>

  <!--SLAM RUN-->
  <node pkg="rviz" type="rviz" name="rviz_nav" output="screen" args="-d $(find mrpt_ekf_slam_3d)/rviz/rviz_conf_ekf_3d.rviz"></node>
  <node pkg="mrpt_ekf_slam_3d" type="mrpt_ekf_slam_3d" name="mrpt_ekf_slam_3d" output="screen"></node>
</launch>

